notebooks/working/investigating_deconfounder.py

```diff
@@ -338,28 +309,39 @@
     ax.axvline(x=obs_ll_per_zi[subject_no], linestyle='--')
 
 plt.show()
```

Comment:
Underneath the plots, can you add some text about the fact that Wang and Blei only justify the use of the mean of the inferred deconfounder (i.e. `z_inferred_mean`) if the posterior of the substitute confounder concentrates at a point.

Feel free to copy their statement directly from their article or put a direct screenshot from their paper (probably even better). Given the images above, two things should be clear to readers:
1. that we are not justified in using the mean of the substitute confounder in our application
2. that this sort of visual analysis (alongside more quantitative analyses such as looking at the variance of `z_inferred` across observations) should be performed to assess convergence-to / concentration-around a point.

To make it even easier, feel free to take from my comments on your other pull request (#68 ) where I first brought this up.